src/model/kaniOutputParser.ts

```diff
@@ -49,6 +49,20 @@ class CheckInstance {
 	}
 }
 
+// Check if stderr and std out contain the error strings or the strings indicating successful compilation and verification
+export function checkOutputForError(outString: string, errString: string): boolean {
+	if (errString.includes('Finished dev') && outString.includes('VERIFICATION:-')) {
```

Comment:
It still feels fragile honestly, I can try to add additional checks but need some ideas @karkhaz .

Reply:
I think this is actually okay. I'm not sure that the if statement below is necessary. I think this entire function could be `return outString.includes('VERIFICATION:-')`